client/scss/components/_preview-panel.scss

```
@@ -117,9 +117,7 @@
     inset-inline-end: 1.5rem;
   }
 
-  &--mobile &__size-button--mobile,
-  &--tablet &__size-button--tablet,
-  &--desktop &__size-button--desktop {
+  .size-button-active {
```

This should be something like

```suggestion
  &__size-button--active {
```

to conform to our BEM practices for CSS.

I've already done something similar in my WIP PR [`adebec8` (#10356)](https://github.com/wagtail/wagtail/pull/10356/commits/adebec86eda1b24020664d3dae82d663ede27ae9) though, so this may not be necessary if that PR gets merged before this one.